frontend/src/pages/LandingPage/LandingPage.tsx

```diff
@@ -1,6 +1,199 @@
+/* eslint-disable react/no-unescaped-entities */
 import React from 'react';
+import drag from './Assets/img/drag-and-drop.png';
```

Comment:
can we move all of the assets in the assets folder? i.e. `frontend/src/assets/LandingPage/`

Reply:
y